membership-attribute-service/app/models/Attributes.scala

```diff
@@ -44,11 +46,12 @@ object Attributes {
 
   implicit val jsWrite: OWrites[Attributes] = (
     (__ \ "userId").write[String] and
-    (__ \ "tier").write[String] and
+    (__ \ "tier").writeNullable[String] and
     (__ \ "membershipNumber").writeNullable[String] and
     (__ \ "adFree").writeNullable[Boolean] and
     (__ \ "cardExpirationMonth").writeNullable[Int] and
-    (__ \ "cardExpirationYear").writeNullable[Int]
+    (__ \ "cardExpirationYear").writeNullable[Int] and
+    (__ \ "Contributor").writeNullable[String]
```

Comment:
This should be lower case "contributor" or "contributorFrequency" as per @johnduffell's comment.